functional_testing/zrobot_templates/framework/ovc_utils/utils.py

```diff
@@ -79,3 +88,13 @@ def get_vm(self, cloudspaceId, vmname):
             if vm['name'] == vmname:
                 return self.ovc_client.api.cloudapi.machines.get(machineId=vm['id'])
         return False
+
+    def wait_for_cloudspace_status(self, cs, status="DEPLOYED", timeout=200):
+        cloudspace = self.get_cloudspace(cs)
```

Comment:
- Better way to write it (alignment is bad)-->
        for _ in range(timeout):
              cloudspace = self.get_cloudspace(cs)
              time.sleep(1)
            if cloudspace["status"] == status:
                return True
         return False